### ISSUE TITLE
detect_client_location on IPv6 machine

### ISSUE THREAD

**Issue by vokac** [PanDAWMS/pilot2#119]
You mention some rucio bug in detect_client_location

https://github.com/PanDAWMS/pilot2/blob/a78652f0d6866d0829e8d0d5bff20af85cecbc8a/pilot/api/data.py#L246

but to be honest I don't know why you force pilot to fail in case this function is not able to resolve address. When I look at rucio master branch

 https://github.com/rucio/rucio/blob/f3784d622e7b8a07635871ec99b31f2a0c1927b0/lib/rucio/common/utils.py#L658

their code never fails and just use some default IP addresses.

Basically your pilot2 and also old pilot always fails on machine without IPv4 address. Could you please update code not to fail in such situation. Unfortunately I really don't know which rucio bug you are trying to address by duplication this function with respect to rucio.common.utils.detect_client_location

**Comment by PalNilsson** [comment]
The bug referred to is ancient and so is the comment; it meant that the function should in fact be pulled into the pilot from the Rucio API. Rucio people can do that for me. The function itself is deprecated and has been updated in both Pilot 1 and 2 dev pilots. The codes will be release asap.
